src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -86,6 +87,57 @@ public static string DownloadSentimentDataset()
         public static string DownloadAdultDataset()
             => Download("https://raw.githubusercontent.com/dotnet/machinelearning/244a8c2ac832657af282aa312d568211698790aa/test/data/adult.train", "adult.txt");
 
+        public static IDataView LoadFeaturizedAdultDataset(MLContext mlContext)
```

Comment:
>LoadFeaturizedAdultDataset [](start = 32, length = 26)

xml doc since this is public

Reply:
done.

---
In reply to: [256623075](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256623075) [](ancestors = 256623075)